pkgs/development/python-modules/anyio/default.nix

```diff
@@ -1,4 +1,5 @@
-{ lib
+{ stdenv
+, lib
```

Comment:
I am feeling a bit nit picky here but stdenv and lib are not really both required.

Reply:
Though I am using both directly, so I don't think it's bad style per se.